faunadb-scala/src/main/scala/faunadb/values/Codec.scala

```diff
@@ -23,6 +29,7 @@ object Decoder {
     def decode(v: Value, path: FieldPath) =
       v match {
         case StringV(s) => Result.successful(s, path)
+        case b: BytesV => Result.successful(b.strValue, path)
```

Comment:
Because we can construct a `BytesV` from a base64 string I thought it would be nice to have an implicit conversion from this type to a string as well